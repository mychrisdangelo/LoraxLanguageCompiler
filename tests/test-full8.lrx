int main()
{
	tree <int>t(2);
	tree <int>t2(2);
	tree <int>t3(1);
	int a;
	int c;
	bool b;
	a = 3;
	c = 4;
	t = 1[2, 3];
	t3 = 9[10[11]];
	t2 = 4[5, 6[7, 8]];
	b = t2 > t;
	print(b);
	b = t2 <= t;
	print(b);

	print(t2 < t, '\n', t >= t2, '\n', t <= t3);

	print('\n', t == t3);
}