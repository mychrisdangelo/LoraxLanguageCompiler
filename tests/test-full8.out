lrxlib.h is being worked on